flycheck.el

```diff
@@ -417,6 +417,45 @@ If set to nil, do not show error tooltips."
   :package-version '(flycheck . "0.25")
   :risky t)
 
+(defcustom flycheck-command-wrapper-function
+  #'identity
+  "Wraps checker commands and its arguments before it gets executed.
+
+The function is usefull to wrap a checker command, if the checker
+is only available in sandbox-like enviroments.  The default value
+does not modify the command or arguments.  In NixOS, checkers are
```

Comment:
I understand that it's important to you, but could you nontheless please shorten the references to NixOS in this docstring to just a simple example?  See, from my point of view NixOS is still just a niche product, and there are many other things this wrapper function can be used for (I'm thinking of `bundle exec` or Cabal sandboxes, etc.).

I'd like to keep the docstring short and to the point.  Instructions or references to very specific systems shouldn't get much place here.